lib/network/modules/SelectionHandler.js

```diff
@@ -656,8 +656,9 @@ class SelectionHandler {
       const hoveredEdgesCount = Object.keys(this.hoverObj.edges).length;
       const hoveredNodesCount = Object.keys(this.hoverObj.nodes).length;
       const newOnlyHoveredEdge = object instanceof Edge && hoveredEdgesCount === 0 && hoveredNodesCount === 0;
+	  const newOnlyHoveredNode = object instanceof Node && hoveredEdgesCount === 0 && hoveredNodesCount === 0;
```

Comment:
This line is not properly indented. It seems to have a `tab` and because of that it looks strange.